BackEnd/administration/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+
+
+def validate_auto_moderation_hours(value: int):
+    if value < 1 or value > 48:
+        raise ValidationError(
+            "Кількість годин має бути в діапазоні 1-48 години"
+        )
+
+
+class AutoModeration(models.Model):
```

Comment:
Are there any strong reasons to have such 1-record model, in comparison with just having something like `AUTO_MODERATION_TIMEOUT` in `settings.py`?

Reply:
I would do it that way, but we want a moderator who is not a programmer and does not have access to the code to be able to change this value. So we want to create an API endpoint and a corresponding UI interface.